api/db/services/dialog_service.py

```
@@ -60,6 +60,28 @@ def _chunk_kb_id_for_doc(row_dict, kb_ids, doc_id):
+                lambda: settings.docStoreConn.search(
+                    ["id"], [],
+                    {"compile_kwd": ["lightgraph"],
+                     "knowledge_graph_kwd": "entity",
+                     "kb_id": [kb_id]},
+                    [], OrderByExpr(), 0, 1,
+                    None, [kb_id]),
+            )
+            if settings.docStoreConn.get_fields(res, ["id"]):
+                return True
+    except Exception:
+        pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)